src/Oro/Bundle/CustomerBundle/Layout/DataProvider/FrontendCustomerUserFormProvider.php

```diff
@@ -125,14 +125,18 @@ private function getCustomerUserFormOptions(CustomerUser $customerUser)
     {
         $options = [];
 
+        if (isset($options['action'])) {
```

Comment:
`$options` was set to an empty array just before that call, action always will be empty. This changes broke the method, now it's doing nothing.

Reply:
if in options['action'] you have an url, you haven't to create a  custom file and extend the provider for only changing the route name